VirtoCommerce.Storefront/Controllers/Api/ApiAccountController.cs

```diff
@@ -110,15 +110,40 @@ public async Task<ActionResult<IdentityResult>> RegisterOrganization([FromBody]
 
             if (ModelState.IsValid)
             {
+                var user = await _userManager.FindByEmailAsync(orgRegistration.Email);
```

Comment:
Can you try to use `await userManager.UserValidators` to enforce more user validation rules?